src/Entities/Player.cpp

```diff
@@ -209,6 +213,15 @@ void cPlayer::Tick(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
 		}
 	}
 
+	Vector3d CurrPos = GetPosition();
+	std::chrono::steady_clock::duration timeDiff = std::chrono::steady_clock::now() - m_LastTickTime;
+	double timeRatio = 1 / (double(timeDiff.count()) * std::chrono::steady_clock::period::num / std::chrono::steady_clock::period::den);
```

Comment:
I really don't like this code as one of the purpose's of std::chrono is to handle this for you. I suspect the best solution is to convert the following code to work with the millisecond count directly.